content.md

```diff
@@ -8,6 +8,8 @@
 4. [How to submit locally saved form data to the remote database](#4-how-to-submit-locally-saved-form-data-to-the-remote-database)
 5. [Different type of data fields and their usage](#5-different-type-of-data-fields-and-their-usage)
 6. [Clearing the application cache](#6-clearing-the-application-cache)
+7. [How to back-up data to google drive](#7-how-to-back-up-data-to-google-drive)
+7. [How to switch on GPS on your device](#8-how-to-switch-on-GPS-on-your-device)
```

Comment:
This link is not working. It needs to be '8-how-to-switch-on-gps-on-your-device'. Best way to find out what it should be is to check the header in the browser using development tools.